static/src/javascripts/lib/$.js

```diff
@@ -9,15 +9,19 @@ bonzo.aug({
     },
 });
 
-const $ = (selector: string, context: ?Element | ?string): bonzo =>
+const $ = (selector: ?string | ?Node, context: ?Element | ?string): bonzo =>
```

Comment:
I think there's some value for compatibility and avoiding null checks everywhere in making this nullable. For instance I believe the target of an event is potentially null, but in that case returning a 0 length bonzo would generally be OK. 

Happy to be argued round on that though.